### PR TITLE
ocicl 2.5.2

### PR DESCRIPTION
**Pull request by atgreen**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?
I get the following error with the brew audit: Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

-----

This release drops the dependency on oras.

### CODE REVIEW

File: Formula/o/ocicl.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class Ocicl < Formula
   desc "OCI-based ASDF system distribution and management tool for Common Lisp"
   homepage "https://github.com/ocicl/ocicl"
-  url "https://github.com/ocicl/ocicl/archive/refs/tags/v2.4.3.tar.gz"
-  sha256 "722ffe7bc0d2559d758f6ebdc803357c53d0fd47612cc498047aea74ca1a481b"
+  url "https://github.com/ocicl/ocicl/archive/refs/tags/v2.5.2.tar.gz"
+  sha256 "c06e0891c9b3d6e58aabeeaac907a100329645a5f73d34797b585046bd3b9c4f"
   license "MIT"
-  revision 2
+  revision 1
```
Comment (chenrui333) [inline Formula/o/ocicl.rb:7]:
```suggestion
```

**Comment by chenrui333** [comment]
also need to squash the commits into one.

**Comment by atgreen** [comment]
There's something wrong with this build.  I'm closing this for now.

**Comment by chenrui333** [comment]
fyi, the bot picked up a new release https://github.com/Homebrew/homebrew-core/pull/194262

**Comment by atgreen** [comment]
Thanks.  I need to debug this. The newer version works when I build it
normally, but not through the homebrew build.

**Pull request closed without merging**